doc/dev.rst

```diff
@@ -249,49 +260,139 @@ What kind of issue is it?
   a new feature proposal or refactoring. Should have steps to reproduce with
   desired and actual behavior.
 
-:code:`help wanted`
-  The core team does not plan to address this issue, perhaps because we don't
-  know how, but we think it would be good to address it. We hope someone from
-  the community will volunteer.
+:code:`refactor`
+  Change that will improve Charliecloud but does not materially affect
+  user-visible behavior.
+
+How important/urgent is it?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:code:`key issue`
-  A particularly important or notable issue.
+Choose *one priority* from:
 
-:code:`question`
-  Support request that does not report a problem or ask for a change. Close
-  these after the question is answered or several days with no activity.
+:code:`high`
+  High priority. If an issue is blocking someone's work it very likely belongs
+  here.
 
-What do we plan to do about it?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:code:`medium`
+  Medium priority.
 
-For all of these, leave other tags in place, e.g. :code:`bug`.
+:code:`low`
+  Low priority. Note: Unfortunately, due to resource limitations, complex
+  issues here are likely to wait a long time, perhaps forever. If that makes
+  you particularly sad on a particular issue, please comment to say why. Maybe
+  it's mis-prioritized.
```

Comment:
Thoughts on including an example for each priority?

Reply:
Discussion 5/19:

* emphasize priority is required
* explain how to choose priority
* who? project lead is responsible for assigning priority, though everyone should give it a shot?

Reply:
I took a crack at this; please take a look when I push.

I hesitate to include examples because I'm concerned about them being representative, i.e., avoid over-generalizing.